system/lang/pl.json

```diff
@@ -39,7 +61,31 @@
     "Extreme": "Ekstremalny",
     "Epic": "Epicki",
     "Stat": "Statystyka",
+    "OracleTable": {
+      "ColumnLabel": {
+        "Roll": "Rzut",
+        "RollRange": "Zakres rzutu",
+        "DiceRollResult": "{dice} wynik rzutu",
+        "Result": "Wynik"
+      }
+    },
     "roll +x": "rzuć +{stat}",
+    "Roll +x": "rzuć +{stat}",
```

Comment:
should be rendered in sentence case

Reply:
What do you mean? :O

Reply:
> What do you mean? :O

here's the original in `en.json`:

```json
"IRONSWORN" : {
    "roll +x": "roll +{stat}",
    "Roll +x": "Roll +{stat}",
}
```
`"roll +x"` is intended for use in situations where it's inserted into another sentence. In English, for example, that means it uses "roll" instead of "Roll", because "roll" is how it would appear when used mid-sentence.

`"Roll +x"` is intended for use as a complete standalone label. It describes a complete phrase (a command). in other words, it's specifically for things that *only* say `"Roll +x"`. So it should be in whatever case would be standard for such a label in a drop-down menu, on a button, and so on. For most languages that use a Latin script (to my knowledge, anyways), this is sentence case (the first letter of the phrase is capitalized, but the rest of the phrase uses standard capitalization).

Reply:
Ah yes! Oh god i didn't notice it 😕. Thank you <3